### DISCUSSION TITLE
Sensitivity threshold for CHIME

### DISCUSSION THREAD

**Discussion by emilyh1701** [chime-frb-open-data/community#32] [Q&A] [answered]
I am currently a student trying to learn more about the CHIME telescope. I am unable to find the sensitivity threshold related to time and position of an FRB. Can you provide me with this information or suggest where I can find it?

Please let me know if you need more information or clarification from me. Thank you.

**Comment by ajosephy** [comment, marked as answer]
Hi @emilyh1701, 

Are you interested in the sensitivity thresholds associated with FRBs already detected by CHIME? Or were you hoping to get the sensitivity threshold for something in the field of view of CHIME, but detected with a different telescope?

If it's the former, we've provided sensitivity thresholds for each burst in the 1st CHIME/FRB catalog.  These thresholds are valid over the entire ~1 year observing period we reported on.

If it's the latter, while it's something we can do (see: https://arxiv.org/abs/2007.02155 for example), there is still some work that needs to go into each case, and the analysis is not something that can easily be made public at this time.  We haven't yet made our full beam model public either, which may be useful in your case.

In summary, I think answering the questions you have is something that will be easier in the future as we make more aspects of the instrument public, but if you can describe your project just a bit more, hopefully I can provide some more useful information.
